File: BackEnd/Zalo_mini/Zalo_mini/Models/sticker_pack.cs
using System;
using System.Collections.Generic;

namespace Zalo_mini.Models
{
    public partial class sticker_pack
    {
        public sticker_pack()
        {
            stickers = new HashSet<sticker>();
        }

        public long id { get; set; }
        public string name { get; set; } = null!;
        public string? description { get; set; }
        public string? thumbnail_url { get; set; }
        public bool? is_free { get; set; }
        public bool? is_active { get; set; }
        public DateTime? created_at { get; set; }

        public virtual ICollection<sticker> stickers { get; set; }
    }
}

File: BackEnd/java/ZaloMini/src/main/java/com/devmam/zalomini/entities/StickerPack.java
package com.devmam.zalomini.entities;

import jakarta.persistence.*;
import lombok.*;
import org.hibernate.annotations.ColumnDefault;
import org.hibernate.annotations.CreationTimestamp;

import java.time.Instant;
import java.util.Date;

@Builder
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Entity(name = "StickerPack")
@Table(name = "sticker_packs")
public class StickerPack {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Long id;

    @Column(name = "name", nullable = false, length = 100)
    private String name;

    @Lob
    @Column(name = "description")
    private String description;

    @Lob
    @Column(name = "thumbnail_url")
    private String thumbnailUrl;

    @ColumnDefault("1")
    @Column(name = "is_free")
    private Boolean isFree;

    @ColumnDefault("1")
    @Column(name = "is_active")
    private Boolean isActive;

    @ColumnDefault("CURRENT_TIMESTAMP")
    @Column(name = "created_at")
    @CreationTimestamp
    @Temporal(TemporalType.TIMESTAMP)
    private Date createdAt;

}